docs/docs/reference/04-astro/index.md

```diff
@@ -87,10 +135,11 @@ const session = await getSession(Astro.request, authOpts)
   <p>Not logged in</p>
 )}
 ```
+### Within the Auth component
 
 Alternatively, you can use the `Auth` component to fetch the session using a render prop.
 
-```tsx title="src/pages/index.astro"
+```astro title="src/pages/index.astro"
```

Comment:
These docs don't support astro syntax highlighting, so we use the next best option for each situation.

```suggestion
```tsx title="src/pages/index.astro"
```